.github/workflows/rabbit_consumer_schedule.yaml

```
@@ -84,4 +84,11 @@ jobs:
           add: ${{ github.workspace }}/OpenStack-Rabbit-Consumer/version.txt
           default_author: github_actions
           committer_name: GitHub Actions
-          committer_email: actions@github.com
\ No newline at end of file
+          committer_email: actions@github.com
+          new_branch: rabbitconsumer_wkflow
+
+      - name: create pull request
+        run: gh pr create -B master -H rabbitconsumer_wkflow --title 'Merge rabbitconsumer_wkflow into master' --body 'Created by Github action'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          
```

Branch name, title and body suggestion
```suggestion
          new_branch: rabbitConsumerDependUpdate

      - name: create pull request
        run: gh pr create -B master -H rabbitConsumerDependUpdate --title 'Merge rabbitConsumerDependUpdate into master' --body 'Updating the dependencies for Rabbit Consumer Image. Created by Github action'
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          
```

---

Very welcome suggestions, added them in.